### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by Zeph180**
- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Add styles to improve the look and feel.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use React Router to make the routes work.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import './App.css';
-import Calculator from './components/Calculator';
+// import Calculator from './components/Calculator';
```
Comment (OLIPLICHE) [inline src/App.js]: - [ ] Hi, Can you remove this commented code to make avoid confusion? Believe me, clean code will make your code more attractive, and disciplined in the eyes of the recruiters.

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+import MyRoutes from './components/MyRoutes';
 
 function App() {
   return (
     <>
-      <Calculator />
+      <MyRoutes />
```
Comment (OLIPLICHE) [inline src/App.js:7]: - Your App is working fine 👍 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Zeph180, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Highlights
- App is working fine👏 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Zeph180,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**